contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -1163,7 +1163,7 @@ define_custom_variables(void)
 							gettext_noop("Temp oid buffer size"),
 							NULL,
 							&temp_oid_buffer_size,
-							0, 0, 131072,
+							65536, 0, 131072,
```

Comment:
should we reenable this GUC in this PR? or in a later one, once fixes like BABEL-4750 are in.

Reply:
I think that we should enable the GUC, at least for development during the current release cycle. If more severe issues are found, we can always disable it prior to cutting the stable branch.